modules/nvidia-gpu-prerequisites.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * architecture/nvidia-gpu-architecture-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="nvidia-gpu-prerequisites_{context}"]
+= NVIDIA GPU prerequisites
+
+* A working OpenShift cluster up and running with a GPU worker node.
+
+* Access to the OpenShift cluster as a `cluster-admin` to perform the necessary steps.
+
```

Comment:
NFD operator must be installed along with an instance of nodefeaturediscovery.  This is required also before you deploy the ClusterPolicy of the GPU Operator.

Reply:
Added.